sources/procfs.go

```diff
@@ -528,8 +528,11 @@ func getJobStatsIOMetrics(jobBlock string, jobID string, promName string, helpTe
 
 func getJobNum(jobBlock string) (jobID string, err error) {
 	jobID = regexCaptureString("job_id: .*", jobBlock)
-	jobID = regexCaptureString("[0-9]*\\.[0-9]+|[0-9]+", jobID)
-	return strings.Trim(jobID, " "), nil
+	jobIDlist := regexCaptureNumbers(jobID)
+	if len(jobIDlist) < 1 {
```

Comment:
Is it bad if this is greater than one? Not saying that it's even possible, just wanting to make sure the code is specifying the right thing.